esp8266-fastled-webserver.ino

```diff
@@ -550,251 +631,247 @@ void loop() {
 
   // insert a delay to keep the framerate modest
   FastLED.delay(1000 / FRAMES_PER_SECOND);
+
+  ArduinoOTA.handle();
```

Comment:
This also enables significant new functionality. This should be a compile-time option, preferably via checking if a symbol is #define'd.